airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceAcceptanceTest.java

```diff
@@ -97,7 +97,7 @@ protected JsonNode getState() {
   }
 
   @Override
-  protected void setupEnvironment(final TestDestinationEnv environment) {
+  protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
```

Comment:
Any chance you could change your testdb methods to rethrow checked exceptions as runtime exceptions? Please